### ISSUE TITLE
speech_to_text_buffered_infer: How to use timestamps 

### ISSUE THREAD

**Issue by MadMaxNice** [NVIDIA/NeMo#8172]
Hello!
How can timestamps be used when using **speech_to_text_buffered_infer_ctc.py** or **speech_to_text_buffered_infer_rnnt.py** ?
Apparently compute_timestamps is not used in the code
Thank you

**Comment by titu1994** [comment]
Sadly those two do not support time stamp calculation. You can use a CTC model with NeMo Forced Aligners to get accurate timestamps

**Comment by GreatHalf** [comment]
It's still not supported, as far as a I can tell.

What's the current recommendation to use all (or most) features of https://github.com/NVIDIA/NeMo/blob/main/examples/asr/transcribe_speech.py when limited in VRAM?

**Comment by eschmidbauer** [comment]
also wondering why we can't do timestamps in speech_to_text_buffered_infer_rnnt.py
it seems like the logits can be returned to calculate timestamps
